docs/operation/bootstrap_backup_restore.rst

```diff
@@ -23,14 +23,34 @@ Restoration procedure
 .. warning::
 
     You cannot use the restore script if you do not have High Availability
-    apiserver because some information required to reconfigure the others
-    nodes are stored in the apiserver.
+    control plane, with at least a 3-node etcd cluster.
 
-.. warning::
+In case of a 3-node etcd cluster (2 nodes + unreachable old bootstrap node)
+you need to remove the old bootstrap node from the etcd cluster before
```

Comment:
We just said it's mandatory, so let's get rid of the "In case of", and reword a bit:

```
Before running the script, the unreachable ``etcd`` member needs to be
unregistered from the cluster. To do so, run the following commands
from a working Node with the ``etcd`` role:

.. code::

   [...]
```

Reply:
This step is mandatory only for a 3 node etcd if you have 4 etcd node you don't need to remove the old one (btw you should remove it I think but it's not needed)